.github/workflows/uat_pre_release_hotfix_argo.yml

```diff
@@ -0,0 +1,272 @@
+name: UAT Pre-Release Hotfix (ArgoCD)
+env:
+  OPENSHIFT_SERVER: ${{ secrets.OPENSHIFT_SERVER }}
+  # service account: gitaction
+  OPENSHIFT_TOKEN: ${{ secrets.OPENSHIFT_TOKEN }}
+  OPENSHIFT_TOOLS_NAMESPACE: "3cd915-tools"
+  MS_TEAMS_WEBHOOK_BUILD_CHANNEL: ${{ secrets.MS_TEAMS_WEBHOOK_URI_BUILD_CHANNEL }}
+  AUTH__KEYCLOAK__SECRET: ${{ secrets.KEYCLOAK_SECRET_UAT }}
+  AUTH__KEYCLOAK__SERVICEACCOUNT__SECRET: ${{ secrets.KEYCLOAK_SERVICEACCOUNT_SECRET }}
+  ASPNETCORE_ENVIRONMENT: "uat"
+
+  ## variables for scripts under git\openshift\4.0\scripts\oc-*.sh
+  APP_PORT: 8080
+  DESTINATION: "uat"
+  OC_JOB_NAME: "test"
+  GIT_URL: "${{github.server_url}}/${{github.repository}}"
+  GIT_BRANCH: "test"
+  APP_NAME: "pims"
+  PROJ_PREFIX: "3cd915"
+  PROJ_TOOLS: "3cd915-tools"
+  PROJ_DEV: "dev"
+  PROJ_TEST: "test"
+  PROJ_PROD: "prod"
+  TAG_DEV: "dev"
+  TAG_TEST: "test"
+  TAG_PROD: "prod"
+  INSTANCE: "-uat"
+  NAMESPACE_OVERRIDE: "3cd915-test"
+  DEPLOYMENT_NAMESPACE: "3cd915-test"
+
+on:
+  workflow_dispatch:
+    inputs:
+      HOTFIX_BRANCH:
+        description: "Enter the name of the branch containing the hotfix"
+        required: true
+
+jobs:
+  ci-cd-start-notification:
+    name: CI-CD Start Notification to Teams Channel
+    runs-on: ubuntu-latest
+    steps:
+      - name: Start notification to Teams Channel
+        uses: dragos-cojocari/ms-teams-notification@v1.0.2
+        with:
+          github-token: ${{ github.token }}
+          ms-teams-webhook-uri: ${{ env.MS_TEAMS_WEBHOOK_BUILD_CHANNEL }}
+          notification-summary: PIMS UAT Pre-Release Hotfix started.
+          notification-color: 17a2b8
+          timezone: America/Los_Angeles
+
+  create-builds:
+    name: create builds
+    needs: ci-cd-start-notification
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Source Code
+        uses: actions/checkout@v4
+      - name: Login to OpenShift
+        uses: redhat-actions/oc-login@v1
+        with:
+          openshift_server_url: ${{ env.OPENSHIFT_SERVER }}
+          openshift_token: ${{ env.OPENSHIFT_TOKEN }}
+          insecure_skip_tls_verify: true
+          namespace: 3cd915-tools
+      - name: create all PSP build configurations for branch
+        shell: bash
+        run: |
+          oc process -f ./openshift/s2i/nginx-runtime/nginx-runtime.yaml -p GIT_REF=${{github.event.inputs.HOTFIX_BRANCH}} -p OUTPUT_IMAGE_TAG="latest-${{github.event.inputs.HOTFIX_BRANCH}}" | oc create --selector ci_cd=true -f -
+          oc process -f ./openshift/4.0/templates/api/build.yaml -p GIT_REF=${{github.event.inputs.HOTFIX_BRANCH}} -p OUTPUT_IMAGE_TAG="latest-${{github.event.inputs.HOTFIX_BRANCH}}" | oc create --selector ci_cd=true -f -
```

Comment:
We would also need a create builds for the new microservices for this to work (or there won't be a correctly named build for the proxy to run). Options here are:
1. add a build.yaml for the proxy and then add a section here to create that build, same as the others.
2. disable this GHA for now, as we always hold UAT until after release these days.

Reply:
This action is for the weird case of a UAT hotfix where we are holding UAT to release to PROD  (I think).
The most used UAT hotfix action is the other one (named uat-hotfix.yaml). I think Devin knows the difference.
This would need to be modified in the near future

Reply:
I think I am going to disable this GHA for now and log tech debt to address this. Otherwise the rest of the changes would be waiting for that. As you said - this is not often used.